djangocms_bootstrap4/contrib/bootstrap4_picture/cms_plugins.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+
+from djangocms_picture.cms_plugins import PicturePlugin
+from djangocms_bootstrap4.helpers import concat_classes
+
+from .models import Bootstrap4Picture
+
+
+class Bootstrap4PicturePlugin(PicturePlugin):
+    """
+    Content > "Image" Plugin
+    https://getbootstrap.com/docs/4.0/content/images/
+    """
+    model = Bootstrap4Picture
+    name = _('Image')
+    change_form_template = 'djangocms_bootstrap4/admin/picture.html'
+    module = _('Bootstrap 4')
+
+    PicturePlugin.fieldsets[0] = (
+        None, {
+            'fields': (
+                'picture',
+                'external_picture',
+                ('picture_fluid', 'picture_rounded', 'picture_thumbnail'),
+            )
+        }
+    )
+
+    fieldsets = PicturePlugin.fieldsets
```

Comment:
This will break the PicturePlugin because we're altering the fieldsets on it to contain fields it does not have. Instead the fieldsets list can be copied and then alterered just for the ``Bootstrap3PicturePlugin``:

```
import copy
class Bootstrap4PicturePlugin(PicturePlugin):
    fieldsets = copy.deepcopy(PicturePlugin.fieldsets)
    fieldsets[0] = (None, {'fields': ......}
```